index.html

```diff
@@ -6,6 +6,10 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+    <form>
+        <label>Card:</label>
+        <input type="text" id ="card-input" title="tooltip" value=" ">
```

Comment:
what will happen when user click Enter? ;)

Reply:
Nothing will happend :) The value of the input is not direct to any of the column at this moment.

Reply:
your page will reload ;) prevent this